sdks/python/apache_beam/testing/util.py

```diff
@@ -128,23 +150,30 @@ def process(self, element, timestamp=DoFn.TimestampParam,
       # the timestamp and window out of the latter.
       return [TestWindowedValue(element, timestamp, [window])]
 
+  class AddWindow(DoFn):
+    def process(self, element, timestamp=DoFn.TimestampParam,
```

Comment:
<!--new_thread; commit:1d8257d6902c54d33062c23cce99e06fbf42c41d; resolved:0-->
You are not using the `timestamp=DoFn.TimestampParam` argument, you can remove it.

Reply:
Done.